### PR TITLE
[IE][VPU]: Implement VPU Ngraph EIP->ReorgYolo transformation

### PR DESCRIPTION
**Pull request by Maxim-Doronin**
### Description
We need to convert ExtractImagePatches op to ReorgYolo to restore the working capacity of myriad plugin while compiling Yolo-v2 models.
It was previously removed in https://github.com/openvinotoolkit/openvino/pull/2687/

### Task
#-42753

### CI
* [ ] private functional tests #-1985
* [ ] dl-benchmark #-522

### CODE REVIEW

File: inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+
+#include <memory>
+#include <vector>
+
+namespace vpu {
+
+ConvertExtractImagePatchesToReorgYoloVPU::ConvertExtractImagePatchesToReorgYoloVPU() {
```
Comment (AndrewBakalinIntel) [inline inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp:15]: Nitpick: I think `VPU` postfix is excessive there, `namespace vpu` is already point us that it is used for VPU.

File: inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        if (extractImagePatches->get_strides() != extractImagePatches->get_sizes()) {
+            return false;
+        }
+
+        const auto& inputPartialShape = extractImagePatches->get_input_partial_shape(0);
+        const auto& sizes = extractImagePatches->get_sizes();
+        const auto& strides = extractImagePatches->get_strides();
+        const auto& rates = extractImagePatches->get_rates();
+
+        // Check that ExtractImagePatches input have static shape and rank == 4
```
Comment (AndrewBakalinIntel) [inline inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp:47]: I think this comment and comments below are excessive, it's very clear from the code what they are actually checking.

File: inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp (hunk 1 above)
Comment (Maxim-Doronin) [reply inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp]: Good catch! Thx! Done

File: inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp (hunk 2 above)
Comment (Maxim-Doronin) [reply inference-engine/src/vpu/common/src/ngraph/transformations/convert_extract_image_patches_to_reorg_yolo_vpu.cpp]: Done

**Review by AndrewBakalinIntel — APPROVED** [review]

**Review by rvyunov — APPROVED** [review]

**Pull request merged**